tests/test_check_datapackage.py

```diff
@@ -0,0 +1,43 @@
+import unittest
```

Comment:
let's call this test_check_schema since the idea is for it to check the schema, rather than the datapackage.